File: C#/PBTKata.Tests/Math/CalculatorProperties.cs
using System;
using FluentAssertions;
using FsCheck;
using FsCheck.Xunit;
using Xunit;
using static PBTKata.Math.Calculator;

namespace PBTKata.Tests.Math
{
    public class CalculatorProperties
    {
        [Property]
        public Property Commutativity(int x, int y)
            => (Add(x, y) == Add(y, x)).ToProperty();

        [Property]
        public Property Associativity(int x)
            => (Add(Add(x, 1), 1) == Add(x, 2)).ToProperty();

        [Property]
        public Property Identity(int x)
            => (Add(x, 0) == x).ToProperty();
    }
}

File: java/src/test/java/math/solution/CalculatorProperties.java
package math.solution;

import com.pholser.junit.quickcheck.Property;
import com.pholser.junit.quickcheck.runner.JUnitQuickcheck;
import org.junit.runner.RunWith;

import static math.Calculator.add;
import static org.assertj.core.api.Assertions.assertThat;

@RunWith(JUnitQuickcheck.class)
public class CalculatorProperties {
    @Property
    public void commutativity(int x, int y) {
        assertThat(add(x, y)).isEqualTo(add(y, x));
    }

    @Property
    public void associativity(int x) {
        assertThat(add(add(x, 1), 1)).isEqualTo(add(x, 2));
    }

    @Property
    public void identity(int x) {
        assertThat(add(x, 0)).isEqualTo(x);
    }
}